src/NavBar.jsx

```diff
@@ -0,0 +1,168 @@
+import React, { Component } from 'react';
+import styled, { keyframes } from 'styled-components';
+import { Link } from 'react-router-dom';
+import './styles/reset';
+
+const MenuContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: center;
+  width: 100%;
+`;
+
+const Nav = styled.nav`
+  display: flex;
+  flex-direction: row;
+  max-height: 3rem;
+  background-color: #383854;
+  width: 100%;
+`;
+
+const Title = styled.h1`
+  display: flex;
+  justify-content: center;
+  border: 2px solid #a3bdff;
+  border-radius: 1rem;
+  color: #a3bdff;
+  font-size: 24px;
+  width: 20rem;
+  font-weight: bold;
+  margin: 1rem;
+  padding: 0 1rem 0.2rem;
+
+  @media only screen and (max-width: 700px) {
+    font-size: 22px;
+    width: 11.5em;
+  }
+
+  @media only screen and (max-width: 400px) {
+    font-size: 22px;
+  }
+`;
+
+const Nav2 = styled.nav`
+  display: flex;
+  align-content: center;
+  flex-direction: row;
+  max-height: 2.5rem;
+  width: 100%;
+`;
+
+const DropdownBtn = styled.a`
+  display: flex;
+  position: absolute;
+  left: 0;
+  align-items: flex-start;
+  color: #a3bdff;
+  margin: 0 0 0 1rem;
+`;
+
+const animnavbar = keyframes`
```

Comment:
this is JavaScript and I think camelCasing variable names is good practice here.